docs/api/components/schemas.yaml

```
@@ -259,35 +259,27 @@ User:
         Mandatory. unless `allow_empty_password` is `true` in the database configs.
       type: string
     admin_channels:
-      description: A list of channels to explicitly grant to the user for the default collection.
+      description: A list of channels to explicitly grant to the user for the default collection. See `collection_access` for channels in named collections.
```

Minor style point, in descriptions I'm trying to standardize on a new line for each sentence, with the literal block style indicator `|-`. (This may seem counterintuitive because `|` is supposed to preserve new lines, but Redocly turns a block of text with a new line at the end of each sentence into a single run-on paragraph, and preserves single blank lines as a blank line between paragraphs.)

```suggestion
      description: |-
        A list of channels to explicitly grant to the user for the default collection.
        See `collection_access` for channels in named collections.
```